### ISSUE TITLE
Password autofill - Kiwi sends browser ID instead of page URL

### ISSUE THREAD

**Issue by bdcarr** [kiwibrowser/src.next#464]
**Describe the bug**
When activating the 1Password autofill dialog from Kiwi, an incorrect list of logins are shown. The dialog says "Select the account you want to use with **Kiwi Browser**", instead of the page URL base like you'd get in other browsers, like Brave or Firefox.

**To Reproduce**
1. [optional, it seems like this should happen regardless of which password mgr you use but I don't know] Install 1Password and set it as your default password manager
2. Create a login entry in 1Password and ensure it has a website URL set (e.g. `reddit.com`)
3. Navigate to that website's login page using Kiwi
4. Tap the username or password field of the login form
5. Tap the 1Password Autofill prompt when it appears
6. Authenticate with 1Password

**Expected behavior**
A list of accounts appears matching the URL base of the page you're on.

**Screenshots**
Sorry for the crap quality, I had to take an actual photo because 1password won't allow screenshots of the autofill dialog. This is from `reddit.com/login`, but you can see the same issue has occurred with four other sites (the only other sites I've tried to log into so far), since they're now associated with Kiwi Browser.

![image](https://user-images.githubusercontent.com/1940488/147795094-548690c4-0329-4ad3-b5b8-3d51a3c7e734.png)

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy A50
 - Android Version : 11
 - Kiwi Browser Version 96.0.4664.46

**Additional context**
I haven't tried it with any other password managers. I have tried it with 1Password on Brave, Chrome and Firefox, and it works properly there.

I read through #64 and #443 but this seems to be a different issue to either of them. #64 has the autofill not working at all, and #443 is about autofill not triggering from the password field.

There was one person [who noted the same issue](https://github.com/kiwibrowser/src.next/issues/64#issuecomment-893453826) in #64, but it didn't get any attention.

**Comment by fredgolightly** [comment]
For me using Bitwarden on many sites (flickr, github, fb, xda, paypal) kiwi will invoke the autofil and launch Bitwarden, but only fill the username not the password. Often it involves selecting the username and password fields repeatedly but these sites never fill the password.

**Comment by Mav3r1x** [comment]
The latest version of kiwi added support for auto filling password api. It works with Google password manager. Thanks @kiwibrowser

**Comment by fredgolightly** [comment]
Ironically this still doesn't work for me in Github login pages, nor nextdns.io login. It will occasionally invoke the password manager ie Bitwarden but now seems to only fill in the username not the password. Same with xda. One site which *does* work virtually all the time is Strava.com, so something must be happening in the login detection setp

**Comment by tortino2** [comment]
yes, on github it doesn't fill the password for me too, but the username works!
any fix yet?

**Comment by shanghaox** [comment]
@kiwibrowser please look at this problem, thanks
